website/docs/guides/core-to-cloud-3.md

```diff
@@ -0,0 +1,196 @@
+
+Both development tools are tailored to suit different audiences and preferences within your team. To streamline your team’s workflow, it's important to know who will prefer the dbt Cloud IDE and who might lean towards the dbt Cloud CLI. This section aims to clarify these preferences.
+
+### dbt Cloud IDE
+A web-based interface for building, testing, running, and version-controlling dbt projects. It compiles dbt code into SQL and executes it directly on your database. The dbt Cloud IDE makes developing fast and easy for new and seasoned data practitioners to build and test changes.
+
+**Who might prefer the dbt Cloud IDE?**
+
+- New dbt users or those transitioning from other tools who appreciate a more guided experience through a browser-based interface.
+- Team members focused on speed and convenience for getting started with a new or existing project.
+- Individuals who prioritize direct feedback from the IDE, such as seeing unsaved changes.
+
+**Key features**
+
+- The dbt Cloud IDE has simplified Git functionality:
+  - Create feature branches from the branch configured in the development environment.
+  - View saved but not-committed code changes directly in the IDE.
+- [Format or lint](/docs/cloud/dbt-cloud-ide/lint-format) your code with `sqlfluff` or `sqlfmt`. This includes support for adding your custom linting rules.
+- Allows users to natively [defer to production](/docs/cloud/about-cloud-develop-defer#defer-in-dbt-cloud-cli) metadata directly in their development workflows, reducing the number of objects.
+- Support running multiple dbt commands at the same time through [safe parallel execution](/reference/dbt-commands#parallel-execution), a [feature](/docs/cloud/about-cloud/dbt-cloud-features) available in dbt Cloud's infrastructure. In contrast, `dbt-core` *doesn't support* safe parallel execution for multiple invocations in the same process.
+
+The dbt Cloud IDE provides a simplified interface that's accessible to all users, regardless of their technical background. However, there are some capabilities that are intentionally not available in the dbt Cloud IDE due to its focus on simplicity and ease of use:
+
+- Pre-commit for automated checks before *committing* code is not available (yet).
+- Mass-generating files / interacting with the file system are not available.
+- Combining/piping commands, such as `dbt run -s (bash command)`, is not available.
+
+### dbt Cloud CLI
+The dbt Cloud CLI allows you to run dbt [commands](/reference/dbt-commands#available-commands) against your dbt Cloud development environment from your local command line. For users who seek full control over their development environment and ideal for those comfortable with the command line.
+
+**Who might prefer the dbt Cloud CLI?**
```

Comment:
added this here @jtcohen6

Reply:
When moving from dbt Core to dbt Cloud, make sure you check the `.gitignore` file contains the [necessary folders](/docs/collaborate/git/version-control-basics#the-gitignore-file). dbt Core doesn't interact with git so dbt Cloud doesn't automatically add or verify entries in the `.gitignore` file. Additionally, if the repository already contains dbt code and doesn't require initialization, dbt Cloud won't add any missing entries to the `.gitignore file`.